platform/solr/solr-schema/src/main/resources/solr/conf/schema.xml

```diff
@@ -57,6 +57,7 @@
 
     <dynamicField name="*_xml" type="string" indexed="false" stored="true" multiValued="true" docValues="false"/>
     <dynamicField name="*_txt" type="string" indexed="true" stored="true" multiValued="true" docValues="true"/>
+    <dynamicField name="*_txt_sort" type="lowercase" indexed="true" stored="false" multiValued="true" docValues="true"/>
```

Comment:
❗ Use a `string` field with `docValues` to improve sorting performance and avoid field cache memory usage.